get_number.rb

```diff
@@ -0,0 +1,48 @@
+# Get My Number Game
+# Written by: you!
+
+puts "Welcome to 'Get My Number!'"
+
+# Получение имени игрока и вывод приветствия.
+print "What's your name? "
+input = gets
+name = input.chomp
+puts "Welcome, #{name}!"
+
+# Сохранение случайного числа.
+puts "I've got a random number between 1 and 100."
+puts "Can you guess it?"
+target = rand(100) + 1
+
+# Отслеживание количества попыток
+num_guesses = 0
+
+# Признак продолжения игры.
+guessed_it = false
+
+until num_guesses == 10 || guessed_it
+
+  puts "You've got #{10 - num_guesses} guesses left."
+  print "Make a guess: "
```

Comment:
rubocop recommends using single-quoted strings in case you don't need string interpolation or special characters.

Reply:
ok